patientInfo/src/main/java/fr/marc/patientInfo/serviceImpl/PatientInfoServiceImpl.java

```diff
@@ -39,17 +40,14 @@ public Iterable<Patient> getPatients() {
 	 * @return the patient according to this id
 	 */
 	@Override
-	public Optional<Patient> getPatientById(Long id) {
+	public Optional<Patient> getPatientById(Integer id){
 		log.info("Get the patient with id = {}",id);
-		/*
 		Optional<Patient> patient = patientInfoRepository.findById(id);
 		if (patient.isPresent()) {
 			return patient;
 		}
 		log.error("There is no patient with id = {} ",id);
-        throw new PatientNotFoundException("There is no patient with id = " + id);
-        */
-		return patientInfoRepository.findById(id);
+		return Optional.ofNullable(new Patient());
```

Comment:
I don't get this line. If there is no patient returned by the service, then we return a new instance? Shouldn't we return **null** instead?

Reply:
J'ai préféré cela à retourner un null de peur que cela ne me génère des problèmes dans la transmission de l'information à Patientui. Dois-je modifier cela ?

Reply:
Pour moi la couche PatientUI doit recevoir un patient qui existe ou sinon null/rien, et donc savoir comment gérer cet état (afficher une erreur si besoin etc)